backstory/src/main/java/com/google/sps/servlets/data/BlobstoreManagerFactory.java

```diff
@@ -0,0 +1,6 @@
+import com.google.sps.servlets.data.BlobstoreManager;
+
```

Comment:
Even though this interface is small, I think you still need the Google license up at the top.